docker-cloud/apps/service-links.md

```diff
@@ -107,6 +107,15 @@ scales up or down. Service links are only updated when a service is deployed or
 redeployed, but are not updated during runtime. No new service link environment
 variables are created when a service scales up or down.
 
+>**Tip:** You can specify one of several [container distribution strategies](/docker-cloud/infrastructure/deployment-strategies.md) for
+applications deployed to multiple nodes. These strategies enable in
+automatic deployments of containers to nodes, and sometimes auto-linking of
+containers. When two services with
+[EVERY_NODE](/docker-cloud/infrastructure/deployment-strategies.md#every-node)
+strategy are linked, containers do not link all-to-all. If a service is linked
```

Comment:
Maybe it is because I am not native but I'd say to clarify:
`If a service with EVERY_NODE strategy is linked to another service with EVERY_NODE strategy, containers will be linked one-to-one on each node`
Feel free to take it or not. 😄